src/exo/LoopIR_scheduling.py

```diff
@@ -3307,16 +3310,28 @@ def map_binop(self, e: LoopIR.BinOp):
         elif e.op == "*":
             if is_const_zero(lhs) or is_const_zero(rhs):
                 return LoopIR.Const(0, lhs.type, lhs.srcinfo)
-            if isinstance(lhs, LoopIR.Const) and lhs.val == 1:
+            if is_const_val(lhs, 1):
```

Comment:
I'm surprised this additional simplification didn't change any golden test outputs. Do we simply not have tests for these cases? If so, we should add them.